sympy/simplify/tests/test_trigsimp.py

```diff
@@ -218,6 +218,10 @@ def test_trigsimp_issue_4032():
 def test_trigsimp_issue_7761():
     assert trigsimp(cosh(pi/4)) == cosh(pi/4)
 
+def test_trigsimp_issue_17633():
+    a,b,c = symbols( 'a,b,c', real = True )
```

Comment:
Do these have to be real?

Reply:
No, I will change that. I works fine without that.